draft-ietf-ppm-dap.md

```diff
@@ -1514,217 +1609,234 @@ to the next phase of the aggregation protocol.
 An `AggregationJobResp` is valid only if it satisfies the following requirement:
 
 * The Helper's `prepare_steps` MUST include exactly the same report IDs in the
-  same order as either the `report_shares` in the Leader's
-  `AggregationJobInitReq` (if this is the first round of continuation) or the
-  `prepare_steps` in the Leader's `AggregationJobContinueReq` (if this is a
-  subsequent round).
+  same order as either the `prepare_steps` in the Leader's
+  `AggregationJobInitReq` (if this is the first step of the aggregation job) or
+  `AggregationJobContinueReq` (if this is a subsequent step).
 
 [[OPEN ISSUE: consider relaxing this ordering constraint. See issue#217.]]
 
 ### Aggregate Continuation {#agg-continue-flow}
 
-In the continuation phase, the leader drives the VDAF preparation of each share
+In the continuation phase, the Leader drives the VDAF preparation of each share
 in the candidate report set until the underlying VDAF moves into a terminal
-state, yielding an output share for all Aggregators or an error. This phase may
-involve multiple rounds of interaction depending on the underlying VDAF. Each
-round trip is initiated by the leader.
+state, yielding an output share for both Aggregators or a rejection. This phase
+may involve multiple HTTP requests from Leader to Helper depending on the
+underlying VDAF.
 
 #### Leader Continuation {#aggregation-leader-continuation}
 
-The Leader begins each round of continuation for a report share based on its
-locally computed prepare message and the previous PrepareStep from the Helper.
-If PrepareStep is of type `failed`, then the leader acts based on the value of
-the `ReportShareError`:
-
-  - If the error is `ReportShareError.report_too_early`, then the leader MAY try
-    to re-send the report in a later `AggregationJobInitReq`.
-  - For any other error, the leader marks the report as failed, removes it from
-    the candidate report set and does not process it further.
-
-If the type is `finished` and the Leader's preparation of this report share is
-also finished, then the report share is aggregated and can now be collected (see
-{{collect-flow}}). If the Leader is not finished, then the report cannot be
-processed further and MUST be removed from the candidate set.
+The Leader begins each step of aggregation continuation with a prep state object
+`state`, which will be `Continued(prep_state)`, and an outbound message
```

Comment:
If the VDAF is 2-round, then `state == Finished(out_share)`. I'd suggest just not asserting the type of `state` here.

```suggestion
`state` and an outbound message
```